lib/src/main/java/org/team100/lib/config/Feedforward100.java

```diff
@@ -35,7 +35,7 @@ public class Feedforward100 {
     private final double kDS;
     private final double staticFrictionSpeedLimit;
 
-    Feedforward100(double kV, double kA, double kSS, double kDS, double staticFrictionSpeedLimit) {
+    private Feedforward100(double kV, double kA, double kSS, double kDS, double staticFrictionSpeedLimit) {
```

Comment:
i think you're not aware of what the "default" visibility does.  if a method is neither "public" nor "private" that means that it is "package private" and can be seen by other classes in the same package.  in this case, that's "org.team100.lib.config," and the most interesting other class in that package is the unit test of this class.  so you ought to put this back like it was.